src/parser/parse_tree_cl.c

```diff
@@ -1191,6 +1234,24 @@ parser_copy_tree (PARSER_CONTEXT * parser, const PT_NODE * tree)
   return copy;
 }
 
+/*
+ * parser_copy_term_dblink () - copies a term for dblink.
+ * 		  It includes the rest of the list pointed to by tree
+ *   return:
+ *   parser(in):
+ *   tree(in):
+ */
+PT_NODE *
+parser_copy_term_dblink (PARSER_CONTEXT * parser, PT_NODE * tree)
+{
+  if (tree)
+    {
+      tree = parser_walk_tree (parser, tree, copy_node_in_term_dblink, NULL, copy_node_in_tree_post, NULL);
```

Comment:
`copy_node_in_tree_post()` is only for PT_SPEC. is it necessary?

Reply:
ok, I will remove it.